blaze-core/src/test/scala/org/http4s/blazecore/util/Http1WriterSpec.scala

```diff
@@ -9,19 +9,23 @@ package blazecore
 package util
 
 import cats.effect._
-import cats.effect.concurrent.Ref
-import cats.effect.testing.specs2.CatsEffect
 import cats.implicits._
 import fs2._
 import fs2.Stream._
-import fs2.compression.deflate
+import fs2.compression.{DeflateParams, deflate}
 import java.nio.ByteBuffer
 import java.nio.charset.StandardCharsets
+
+import cats.effect.std.Dispatcher
 import org.http4s.blaze.pipeline.{LeafBuilder, TailStage}
 import org.http4s.util.StringWriter
+
 import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits._
 
 class Http1WriterSpec extends Http4sSpec with CatsEffect {
+  implicit val d: Dispatcher[IO] = Dispatcher[IO].allocated.unsafeRunSync()._1
```

Comment:
This is WIP. We need to call the `_2` function after the tests.

Reply:
We have a `withResource` in `Http4sSpec` that I think would work.

Reply:
https://github.com/http4s/http4s/pull/3894/commits/1976730b781568a2ef3769e2a4fa936aecc8974d